crates/anvil-polkadot/tests/it/standard_rpc.rs

```diff
@@ -174,3 +179,672 @@ async fn test_send_to_uninitialized() {
     );
     assert_eq!(alith_final_balance_2, alith_final_balance + transfer_amount);
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_estimate_gas() {
+    let anvil_node_config = AnvilNodeConfig::test_config();
+    let substrate_node_config = SubstrateNodeConfig::new(&anvil_node_config);
+    let mut node = TestNode::new(anvil_node_config.clone(), substrate_node_config).await.unwrap();
+    let alith = Account::from(subxt_signer::eth::dev::alith());
+    let baltathar = Account::from(subxt_signer::eth::dev::baltathar());
+    let transfer_amount = U256::from_str_radix("100000000000000000", 10).unwrap();
+    let transaction = TransactionRequest::default()
+        .value(transfer_amount)
+        .from(Address::from(ReviveAddress::new(alith.address())))
+        .to(Address::from(ReviveAddress::new(baltathar.address())));
+
+    let estimated_gas: pallet_revive::U256 = unwrap_response(
+        node.eth_rpc(EthRequest::EthEstimateGas(
+            WithOtherFields::new(transaction.clone()),
+            None,
+            None,
+            None,
+        ))
+        .await
+        .unwrap(),
+    )
+    .unwrap();
+    let tx_hash = node.send_transaction(transaction, None).await.unwrap();
+    unwrap_response::<()>(node.eth_rpc(EthRequest::Mine(None, None)).await.unwrap()).unwrap();
+    tokio::time::sleep(std::time::Duration::from_millis(400)).await;
+    let receipt = node.get_transaction_receipt(tx_hash).await;
+
+    assert!(estimated_gas > receipt.gas_used);
```

Comment:
nit: can we link some reference based on which we can understand why we assert this way rather than equality?